README.md

```diff
@@ -110,7 +152,20 @@ def mystery6(s)
 end
 ```
 
-- What is mystery6("goodnight moon")?
+- What is mystery6("goodnight moon")?  
+     " moon goodnight" (note: includes initial underscore)
 - What is mystery6("Ada Developers Academy")?
+    " Academy Developers Ada"
 - What is mystery6("Hi, there!")?
+    " there! Hi,"
+
 - _Added Fun: How could we make the reversal happen by letter, instead of by word (i.e. Make it so that mystery6("goodnight moon") returned "noom thgindoog")?_
+
+
+def mystery6B(s)
+  if s == nil || s.length == 0
+    return ""
+  else
+    return mystery6B(s[1..-1]) + s[0]
+  end
+end
```

Comment:
+1 my solution too. Though I wrote the base case differently, they are essentially the same. I didn't account for nil though.